kustomize/base/quay.deployment.yaml

```diff
@@ -129,3 +141,7 @@ spec:
             - name: extra-ca-certs
               readOnly: true
               mountPath: /conf/stack/extra_ca_certs
+            - name: postgres-certs
+              mountPath: /run/secrets/postgres
+            - name: postgres-certs-store
+              mountPath: /.postgres
```

Comment:
this should be `/.postgresql` to match what's defined in [quay-entrypoint](https://github.com/quay/quay/blob/master/conf/init/client_certs.sh#L3)